daemon/daemon.go

```diff
@@ -11,14 +11,44 @@ import (
 )
 
 type Daemon interface {
-	Name() string       // Name of daemon
-	Path() string       // absolute path to the binary
-	Args() string       // args to pass to binary during start
-	SocketFile() string // path to socket file to connect to
-	PidFile() string    // path to pidfile
+
+	// Name of the daemon. This will be primarily used for logging.
```

Comment:
Godoc documentation generally follows a '<Identifier> does something' format. So this would be documented as `// Name should return the name of the daemon. This will be primarily used for logging.` Change all the other comments to follow this as well.

`golint` should have caught this, but I've just realised that since it's being run as a part of `gometalinter` with warnings silent, you are not being warned. I'll fix that later. For now, you can run `golint  <file>` to check if the file is good.